dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -1080,7 +1080,7 @@ RtpsUdpDataLink::RtpsWriter::customize_queue_element_helper(
     max_sn_ = std::max(max_sn_, seq);
     if (!durable_ && !is_pvs_writer() &&
         element->subscription_id() == GUID_UNKNOWN &&
-        previous_max_sn != max_sn_.previous()) {
+        previous_max_sn < max_sn_.previous()) {
```

Comment:
The implication is that `previous_max_sn` > `max_sn_.previous()` is possible.  I would like to understand how this may occur.  If it is the for case of `previous-max_sn == max_sn` then that is probably okay.

Reply:
Yeah, that's the only issue I saw, when they were equal, then previous_max_sn != max_sn_.previous() would be true and it would generate an inverted gap.